### DISCUSSION TITLE
How to use custom Github Action for grading and set number of points ?

### DISCUSSION THREAD

**Discussion by neurorepro** [community/community#36575] [GitHub Education]
Hi,

I'm starting with Github Classroom, and i'd like to know how to set grading tests using Github actions (instead of autograding GUI), and how to set number of points for each of these tests ?

Any help would be much appreciated.

**Comment by ossanna16** [comment]
Hey there @neurorepro and thank you for asking a good question! I'm glad you found our community 🙂
